fe/fe-core/src/main/java/org/apache/doris/datasource/jdbc/client/JdbcClient.java

```diff
@@ -409,6 +409,8 @@ protected List<String> filterDatabaseNames(List<String> remoteDbNames) {
     protected Set<String> getFilterInternalDatabases() {
         return ImmutableSet.<String>builder()
                 .add("information_schema")
+                .add("performance_schema")
```

Comment:
Why filter performance_schema?

Reply:
We do not need to obtain MySQL's performance_schema in Doris, which will increase the burden of obtaining metadata, so filtering